shell/platform/windows/settings_plugin.cc

```diff
@@ -55,6 +57,12 @@ void SettingsPlugin::SendSettings() {
 }
 
 void SettingsPlugin::StartWatching() {
+  RegOpenKeyEx(HKEY_CURRENT_USER, kGetPreferredBrightnessRegKey,
+               RRF_RT_REG_DWORD, KEY_NOTIFY, &preferred_brightness_reg_hkey_);
+  RegOpenKeyEx(HKEY_CURRENT_USER, kGetTextScaleFactorRegKey, RRF_RT_REG_DWORD,
+               KEY_NOTIFY, &text_scale_factor_reg_hkey_);
```

Comment:
Could we make `preferred_brightness_reg_hkey_` and `text_scale_factor_reg_hkey_` protected and then test that these are non-null after calling `StartWatching`?

Reply:
After the lines, those variables are checked whether they aren't null, and it affects whether WatchPreferredBrightnessChanged and WatchTextScaleFactorChanged is called.
Without this fix, the new test which checks they are called will fail.

Reply:
Ah gotcha, perfect!